BSPConvert.Lib/Source/EntityConverter.cs

```diff
@@ -209,6 +213,80 @@ private void ConvertFuncStatic(Entity funcStatic)
 			funcStatic.ClassName = "func_brush";
 		}
 
+		private void ConvertFuncPlat(Entity entity)
+		{
+			var moveDistance = 0f;
+			var brushThickness = GetBrushThickness(entity);
+
+			if (float.TryParse(entity["height"], out var height))
+				moveDistance = height + brushThickness;
+			else if (float.TryParse(entity["lip"], out var lip))
+				moveDistance = -(lip - 2 - (brushThickness * 2));
+
+			if (string.IsNullOrEmpty(entity.Name))
+			{
+				entity.Name = $"plat{platIndex}";
```

Comment:
Renaming the entity will make it so it does not get added to the `entityDict` properly. It's only a problem if the func_plat is the target of another entity.

Also is there a way to give the platform a unique name without adding the `platIndex` class level variable? Maybe you could use the index of the entity in the `q3Entities` collection. If not, it's not a significant concern.